commands/command.go

```diff
@@ -195,21 +196,24 @@ func listCommandCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		teams = getTeamsFromTeamArgs(c, args)
 	}
 
+	var errs *multierror.Error
 	for i, team := range teams {
 		if team == nil {
 			printer.PrintError("Unable to find team '" + args[i] + "'")
+			errs = multierror.Append(errs, fmt.Errorf("unable to find team '%s'", args[i]))
 			continue
 		}
 		commands, _, err := c.ListCommands(team.Id, true)
 		if err != nil {
 			printer.PrintError("Unable to list commands for '" + team.Id + "'")
+			errs = multierror.Append(errs, fmt.Errorf("unable to list commands for '%s'", team.Id))
```

Comment:
```suggestion
			errs = multierror.Append(errs, fmt.Errorf("unable to list commands for '%s': %w", team.Id, err))
```
We should add the original error here.